### PR TITLE
Ignore python version save file when using pyenv

### PR DESCRIPTION
**Pull request by fredvd**
for python version management as a developer/contribuor to docs.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@
 pyvenv.cfg
 /_build
 /styles/Microsoft
+.python-version
```
Comment (stevepiercy) [inline .gitignore:11]: This should go in the user's gitignore, with whatever implementation they choose.

This `.gitignore` should include only those things that are generated by the project, not things that the user may or may not use.

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (fremwilplone) [reply .gitignore:11]: https://github.com/plone/buildout.coredev/blob/6.0/.gitignore

Comment (fredvd) [reply .gitignore:11]: https://github.com/plone/buildout.coredev/blob/6.0/.gitignore

Comment (fredvd) [reply .gitignore:11]: https://github.com/plone/volto/blob/master/.gitignore

Comment (stevepiercy) [reply .gitignore:11]: That's fine for those repos, but not this one.

**Comment by stevepiercy** [comment]
I'm closing this as "won't fix".

**Pull request closed without merging**